apis/core/hydra/index.ttl

```diff
@@ -0,0 +1,15 @@
+@prefix hydra: <http://www.w3.org/ns/hydra/core#> .
+@prefix code:  <https://code.described.at/> .
+
+<api> a                    hydra:ApiDocumentation ;
+      hydra:entrypoint     <> ;
+      hydra:supportedClass hydra:Resource,
+                           hydra:Collection .
+
+hydra:Resource hydra:supportedOperation [ hydra:method       "GET" ;
+                                          code:implementedBy [ a         code:EcmaScript ;
+                                                               code:link <node:@hydrofoil/labyrinth/resource#get> ] ] .
+
+hydra:Collection hydra:supportedOperation [ hydra:method       "GET" ;
+                                          code:implementedBy [ a         code:EcmaScript ;
+                                                               code:link <node:@hydrofoil/labyrinth/collection#get> ] ] .
```

Comment:
I don't really like this turtle formatting because:
- It's really hard to do by hand
- Changing something in one line may require changing all the other lines of the resource (= terrible diffs)
- It doesn't help reading

Do we have linting/formatting rules for turtle?

Reply:
What do you propose?

I get this formatting by a WebStorm plugin. It's not great but at least I get consistent structure. Problem is that not everyone uses webstorm.

I wish there was an eslint turtle parser+formatter. Otherwise we're back to manually formatting which can only be worse.

Reply:
> at least I get consistent structure

The line above is not even consistent, `code:implementedBy` is not aligned correctly ;)

My proposition would be something like:
```
hydra:Collection hydra:supportedOperation [ 
  hydra:method  "GET" ;
  code:implementedBy [ 
    a  code:EcmaScript ;
    code:link <node:@hydrofoil/labyrinth/collection#get> ;
  ] ;
] ;
.
```

Reply:
Heh yea, not consistent because it does not format itself unless reformatted in the IDE 😬.

Thus my question, I'm fine with tweaking the desired formatting but how do we keep it that way? I don't want to spend time doing that manually all the time

Reply:
I don't have an automated solution, but I think we should at least chose a format that is doable by hand